### ISSUE TITLE
Add link-to-search on "Package not found" page

### ISSUE THREAD

**Issue by lspitzner** [haskell/hackage-server#440]
i have a shortcut in browser to navigate to specific package. when i make error typing/remembering name, it would be nice if i'd get a link to search for this package. I.e.

```
https://hackage.haskell.org/packages/foo
->
https://hackage.haskell.org/packages/search?terms=foo
```

(Similar to wikipedia's second link, see https://en.wikipedia.org/wiki/Haksell)

**Comment by dcoutts** [comment]
The approach here would be to use a different error page template and then make that template provide a link to the search, with the term filled out.

**Comment by gbaz** [comment]
@SooryaN this looks like another ticket that may relate to your work?

**Comment by gbaz** [comment]
Should be fixed by merge of #551
